elasticsearch/probe/readiness.sh

```diff
@@ -20,7 +20,7 @@
 ES_REST_BASEURL=https://localhost:9200
 EXPECTED_RESPONSE_CODE=200
 secret_dir=/etc/elasticsearch/secret
-max_time=${max_time:-1}
+max_time=${max_time:-0.8}
```

Comment:
By "take down" I mean the Service would stop routing requests to the pods that failed the readiness probe.